routes.js

```diff
@@ -1,86 +1,86 @@
-// const express = require('express');
-// const createMyCustomApi = express.Router();
-// const fs = require('fs');
+const express = require('express');
+const createMyCustomApi = express.Router();
```

Comment:
As a rule of thumb, please use verbs like `create , make , get` for function names. As they are doing some actions. And use nouns to name variables , instances of objects, objects, like `user, customer, memeber`.
In this case, this is the instance of a router object, so better name would be just `myCustomAPI` or `membersAPI` or `memebersRouter`. Please pick the one you like more and rename this variable

Reply:
Done